Users/Install.rst

```diff
@@ -199,7 +199,7 @@ application also asks for the packages needed in case it does not
 have it.
 
 The requirements files ( Gemfile, requirements.txt, etc.) are in
-the coala-bears repository and you should not get them from 
+the coala-bears repository and you should not get them from
```

Comment:
gitmate has a bug which made this possible, the coala 0.9.1 release is needed to fix that (and we should run coala on this repo since gitmate is incremental!)